src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -295,70 +323,127 @@ function Setup({ origin }: { origin: KeylessBackupOrigin }) {
 
   switch (keylessBackupStatus) {
     case KeylessBackupStatus.InProgress: {
-      return renderInProgressState(t('keylessBackupStatus.setup.inProgress.title'))
+      return (
+        <SafeAreaView style={styles.safeAreaView}>
+          {isOnboarding && (
+            <CustomHeader
+              title={
+                <HeaderTitleWithSubtitle
+                  title={t('keylessBackupStatus.setup.inProgress.title')}
+                  subTitle={t('registrationSteps', { step, totalSteps })}
+                />
+              }
+              style={styles.header}
+            />
+          )}
+          <ScrollView contentContainerStyle={styles.bodyContainer}>
+            <View style={iconMarginTop}>
+              <GreenLoadingSpinner />
+            </View>
+            {!isOnboarding && (
+              <Text style={styles.title}>{t('keylessBackupStatus.setup.inProgress.title')}</Text>
+            )}
+          </ScrollView>
+        </SafeAreaView>
+      )
     }
     case KeylessBackupStatus.Completed: {
       return (
-        <SafeAreaView style={styles.container}>
-          <View style={styles.finishedContainer}>
-            <View style={styles.iconContainer}>
+        <SafeAreaView style={styles.safeAreaView}>
+          {isOnboarding && (
+            <CustomHeader
+              title={
+                <HeaderTitleWithSubtitle
+                  title={t('keylessBackupStatus.setup.completed.title')}
+                  subTitle={t('registrationSteps', { step, totalSteps })}
+                />
+              }
+            />
+          )}
+          <ScrollView contentContainerStyle={styles.bodyContainer}>
+            <View style={iconMarginTop}>
               <GreenLoadingSpinnerToCheck />
             </View>
-            <Text style={styles.title}>{t('keylessBackupStatus.setup.completed.title')}</Text>
+            {!isOnboarding && (
+              <Text style={styles.title}>{t('keylessBackupStatus.setup.completed.title')}</Text>
+            )}
             <Text style={styles.body}>{t('keylessBackupStatus.setup.completed.body')}</Text>
-          </View>
+          </ScrollView>
           <Button
             testID="KeylessBackupProgress/Continue"
             onPress={onPressContinue}
             text={t('continue')}
             size={BtnSizes.FULL}
             type={BtnTypes.PRIMARY}
-            style={styles.button}
             touchableStyle={styles.buttonTouchable}
+            style={[
+              styles.buttonContainer,
+              isOnboarding ? insetsStyle : { marginBottom: Spacing.Thick24 },
```

Comment:
wonder if we could drop the conditional padding and just be consistent for both settings and onboarding

Reply:
Removed in 9886de417fa3605fba501d79e259b17ab1544fe7!